packages/polymath-issuer/src/utils/parsers/index.js

```diff
@@ -131,7 +131,11 @@ export function parseWhitelistCsv(file: string) {
             return false;
           }
           if (value.split('/').length === 3) {
-            return moment(value, ['MM/DD/YY', 'MM/DD/YYYY']).toDate();
```

Comment:
If the user is uploading a file that has 2-digit year date it should be parsed, right @RafaelVidaurre ? Cause the PR I submitted handles the case of 2-year digits.

Reply:
I don't think we should allow 2-digit years since our upload instructions refers to a MM/DD/YYYY format and our sample mint file also has 4-digits for the year.